app/helpers/ontology_search_helper.rb

```diff
@@ -0,0 +1,20 @@
+module OntologySearchHelper
+
+  def render_filter(klass, include_blank='', selected_key: nil)
```

Comment:
Surrounding space missing in default value assignment.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.